projects/baby_driver/scripts/message_defs.py

```diff
@@ -17,3 +17,7 @@ class BabydriverMessageId:
     """
 
     STATUS = 0
+
+    #gpio_get constants
```

Comment:
nitpick: please please always put a space after the `#` in a comment, though this one is unnecessary